src/app/pages/CpsPgl/index.jsx

```diff
@@ -46,6 +47,11 @@ const CpsPglContainer = ({ pageData }) => {
     ['relatedContent', 'groups', 0, 'promos'],
     pageData,
   );
+  const indexImagePath = path(['promo', 'indexImage', 'path'], pageData);
+  const indexImageLocator = indexImagePath
+    ? getImageParts(indexImagePath)[1]
+    : null;
```

Comment:
Can these methods be extracted into a separate util, similar to src/app/containers/ArticleMain/utils.js (although I am not keen on the fact that it is within ArticleMain, and is used for CPS pages too).